diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/EmptyBlockWarnTest.kt

```diff
@@ -125,24 +139,9 @@ class EmptyBlockWarnTest : LintTestBase(::EmptyBlock) {
         lintMethod(
             """
                     |fun foo() {
-                    |   val y = listOf<Int>().map {} 
+                    |   val y = listOf<Int>().map { } 
                     |}
-                """.trimMargin(),
-            LintError(2, 30, ruleId, "${EMPTY_BLOCK_STRUCTURE_ERROR.warnText()} empty blocks are forbidden unless it is function with override keyword", false)
-        )
-    }
-
-    @Test
-    @Tag(WarningNames.EMPTY_BLOCK_STRUCTURE_ERROR)
-    fun `check empty lambda with config`() {
```

Comment:
Why did you remove this test, it was checking behavior with different configuration flag